.github/workflows/validate-pr-python.yaml

```diff
@@ -93,6 +93,13 @@ jobs:
       - name: Test
         run: pipenv run test-ci
 
+      - uses: mikepenz/action-junit-report@v4.3.1
```

Comment:
Boyscout rule: report these test results to the run summary so that we don't have to dive into the logs every time.